adapters/taboola/taboola.go

```diff
@@ -35,9 +39,17 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		return nil, []error{err}
 	}
 
+	requestEndpoint := a.endpoint
+	if len(request.Imp) > 0 {
+		if request.Imp[0].Native != nil {
+			requestEndpoint = NATIVE_ENDPOINT
```

Comment:
Note that you determine whole request type just by the first impression. It may be a potential source of bugs if there are impressions with different media types in one request. However this is not a likely case. Optional suggestion: split request to sub requests if there are imps with different types and use corresponding  endpoint urls.

Reply:
I would also like to request if you could add a test where request has multiple impressions having at least one native and banner bid type.

Reply:
@VeronikaSolovei9  @gargcreation1992 
we mentioned in the documentation that currently we don't support **multi-formate requests.** 
However, we already added such a test only for checking the adapter functionality with multi-format requests (our server will respond with no-bid)
the unit test file name is: `multiImpressionsRequest.json`

Reply:
`multiFormatImpressionsRequest.json`